packages/create-sitecore-jss/src/templates/react/scripts/scaffold-component.js

```diff
@@ -38,6 +39,11 @@ if (fs.existsSync(componentManifestDefinitionsPath)) {
 
 const componentOutputPath = scaffoldComponent();
 
+// Component is not scaffolded if it already exists
+if (!componentOutputPath) {
+  return;
+}
+
 console.log();
 console.log(chalk.green(`Component ${componentName} has been scaffolded.`));
```

Comment:
The success message has been moved to scaffoldFile(), so this log can be removed